### PR TITLE
chore: 3.4.1.0 changelog

### PR DESCRIPTION
**Pull request by acgoldsmith**
### Description

Add changelog for 3.4.1.0 release.

DOCU-3403
### Testing instructions

Preview link: https://deploy-preview-6186--kongdocs.netlify.app/gateway/changelog/#3410

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by acgoldsmith** [comment]
@outsinre , a few questions:
* are there other dependencies like bumped versions to include? 
* the graphQL fix that's listed as a breaking change: what's the customer impact?
* for the new expressions route feature, where can I find the docs that explain how to use? I checked https://github.com/Kong/kong/pull/11348 but didn't see there.

**Comment by outsinre** [comment]
> @outsinre , a few questions:
> 
>     * are there other dependencies like bumped versions to include?
> 
>     * the graphQL fix that's listed as a breaking change: what's the customer impact?
> 
>     * for the new expressions route feature, where can I find the docs that explain how to use? I checked [feat(router): support http queries in expression kong#11348](https://github.com/Kong/kong/pull/11348) but didn't see there.

Hello @acgoldsmith ,

1. Please refer to https://github.com/Kong/kong-ee/pull/6649/ or https://github.com/Kong/kong-ee/blob/next/3.4.x.x/changelog/3.4.1.0/3.4.1.0.md on `3.4.1.0` changelog.
2. Regarding https://github.com/Kong/kong/pull/11348, I will ping @chronolaw to add a new user-facing doc PR.

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -9,6 +9,75 @@ Changelog for supported Kong Gateway versions.
...
+  * Remove references from rotation in case their configuration has changed
+* Tracing: fixed an issue that resulted in some parent spans to end before their children due to different precision of their timestamps.
+ 
+#### Plugin
+
+* [**OpenTelemetry**](/hub/kong-inc/opentelemetry/) (`opentelemetry`): fix an issue that resulted in invalid parent IDs in the propagated tracing headers
+* [**mTLS Authentication**](/hub/kong-inc/mtls-auth/) (`mtls-auth`): should not cache the network failure when doing revocation check
+* [**Canary**](/hub/kong-inc/canary/) (`canary`): allow the `start` field to be a time that occurs in the past.
+* [**SAML**](/hub/kong-inc/saml/) (`saml`): When the redis session storage is incorrectly configured, users now receive a 500 error instead of being redirected endlessly.
+* [**OpenID Connect**](/hub/kong-inc/openid-connect/) (`openid-connect`): Fix issue on token revocation on logout where the code was revoking refresh token when it was supposed to revoke access token when using the discovered revocation endpoint.
```
Comment (Guaris) [inline app/gateway/changelog.md:68]:
```suggestion
* [**OpenID Connect**](/hub/kong-inc/openid-connect/) (`openid-connect`): Fix the issue on token revocation on logout where the code was revoking the refresh token when it was supposed to revoke access token when using the discovered revocation endpoint.
```

**Comment by outsinre** [comment]
@acgoldsmith This is @chronolaw 's router user-facing PR: https://github.com/Kong/docs.konghq.com/pull/6201

**Pull request merged**